web/src/user_profile.js

```diff
@@ -109,6 +112,18 @@ function render_user_group_list(groups, user) {
     });
 }
 
+function render_manage_profile_content(user) {
+    $("#user-profile-modal .modal__body").css("height", "52vh");
```

Comment:
I'm interested a) in where `52vh` is coming from, much like the `60vh` value elsewhere in this PR. That should be documented even by a comment, but I would also be more comfortable seeing CSS information like this preserved in CSS files and accessed via a class.

Reply:
I have given the explanation for this in the commit description, do let me know if you have any other thoughts or a better alternative to this

Reply:
That's good, but there should be an explanation in the code here--if someone were to need to modify this in the future, don't make them hunt down a commit message :)

Reply:
Sure, that was just a draft with some changes. I wanted to get an early review, so I didn't finish the commits or the comments. I'll do so in the future :)